### ISSUE TITLE
[Feature] Make portable temp reg not consume power when near block temp reg

### ISSUE THREAD

**Issue by benknight04861** [p455w0rd/ToughExpansion#8]
Tough Expansion version 1.11.2-3.0.9

I have a charged portable temperature regulator on me and a standard powered block temperature regulator in the space with me.  I know that the block temperature regulator is working because if I take off my portable regulator I still don't overheat or get cold.  However whenever I have the portable regulator on me (even in the regulated space) the portable regulator still uses power.

TLDR - When I am near a functional block temperature regulator I don't expect my personal temperature regulator to use power.

**Comment by p455w0rd** [comment]
This is how it's supposed to work. I don't see the logic here. It's comparable to having a generator outside of your house and expecting the battery in your phone to not consume any power...

**Comment by p455w0rd** [comment]
I've decided to make this a config option though, because I highly doubt you are the only one that feels this way. :) Please don't ask for ETA, it will likely be a bit as I am currently fully consumed with projects, but it's on my todo list.

**Comment by benknight04861** [comment]
Thanks for keeping the idea.  In case it helps you understand why I'd ask in the first place, I see the analogy more like not needing to wear a winter coat when in a heated home.  But maybe I'm misunderstanding how TaN and your expansion handle temperatures?  Anyway, thanks again and no rush.  I'm just glad people like you make these mods and maintain them at all. :- )
